### ISSUE TITLE
Is progress on FSharpLint as slow as it seems?

### ISSUE THREAD

**Issue by cmeeren** [fsprojects/FSharpLint#576]
I mean no criticism; I am just trying to take stock of the state of this tool.

Based on the open issues, the commit frequency, and the numerous open PRs, it seems like FSharpLint development is very slow. For example, I see that the "v1 issue" #434 was posted almost one and a half years ago.

I would like to use a linter for F# to improve code quality (particularly for keeping complexity in check, think cyclomatic complexity, the number of used parameters/variables/members in a function, etc.). FSharpLint seems to be the only alternative, so I gave it a try. However, I quickly encountered quite a few bugs that rendered many checks unusable (many of these bugs have been reported previously), and some blockers like FSharpLint failing outright with no message and a non-helpful stack trace for certain projects I can't easily produce minimal repros for. There are also some rules I'd like to see implemented, but given the seeming lack of activity in handling issues and PRs, my motivation for any kind of contribution (even just posting issues) is fairly low.

I am posting this because I just wanted to get the view of the maintainers of FSharpLint, in case my view of the situation is incorrect.

Again, I want to stress that I mean no criticism. As an OSS maintainer myself, I fully understand that this is (I assume) a hobby project with limited resources available for development and maintenance, and that the maintainers may not have the resources to actively work on this project.

**Comment by knocte** [comment]
> Is progress on FSharpLint as slow as it seems?

I only recently joined the maintainership team. In my view it is, but not because of me, and let me elaborate on that: the amount of PRs we get is very low and I try to quickly take action on them. The PRs that you still see open without feedback are not really PRs from volunteers, they are PRs that my team from my company has put together as an effort from the tooling side of things. As such, we will merge them when we feel they are ready, and they take a while because tooling is not a priority of the business. (FTR, devs from my team are/were janus, webwarrior-ws, parhamsaremi and su8898.) We have even fixed bugs reported by the community on company time (example: bug 555)!

With this, I hope I clarified your fears and you provide us with bug reports (and hopefully PRs) soon.
